tests/system/conftest.py

```diff
@@ -27,20 +27,34 @@
 from pytest import CallInfo, ExitCode, Function, TestReport
 
 
+def missing_kubeclient(self):
+    raise AttributeError("Kubeclient was not setup and is unavailable")
+
+
 def pytest_sessionstart(
     session: pytest.Session,
 ):
+    kubeconfig_content = None
     # Setup K8S client for use in system tests.
-    base64_kubeconfig_content = os.environ.get("SYSTEM_TEST_KUBECONFIG")
-    if not base64_kubeconfig_content:
-        raise ValueError("The Kubeconfig for the system test session was not provided.")
-    kubeconfig_content = base64.b64decode(base64_kubeconfig_content)
-    with NamedTemporaryFile() as tempfile:
-        tempfile.write(kubeconfig_content)
-        config.load_kube_config(
-            config_file=tempfile.name,
-        )
-        session.kube_client = k8s_client.CoreV1Api()
+    try:
+        base64_kubeconfig_content = os.environ["SYSTEM_TEST_KUBECONFIG"]
+        kubeconfig_content = base64.b64decode(base64_kubeconfig_content)
+    except (ValueError, KeyError):
+        print("Kubeconfig was empty or invalid.")
```

Comment:
1. Print the exception error message?
2. Any chance to use a logger here?

Reply:
fixed